### PR TITLE
V3

### PR DESCRIPTION
**Pull request by lemoustachiste**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,6 +20,7 @@ data/work/*
 data*/*
 dist/*
 build/*
+node_modules/*
```
Comment (lemoustachiste) [inline .gitignore:23]: not sure if relevant in this case

File: cert_issuer/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '2.0.28'
+__version__ = '3.0.0b2'
```
Comment (lemoustachiste) [inline cert_issuer/__init__.py:1]: make it 3.0.0

File: requirements.txt [hunk 3]
```diff
@@ -1,13 +1,14 @@
-cert-core>=2.1.10
-cert-schema>=3.0.0a9
-chainpoint>=0.0.2
+cert-core>=3.0.0b1
+cert-schema>=3.0.0b4
```
Comment (lemoustachiste) [inline requirements.txt:2]: publish v3 and update here

File: requirements.txt [hunk 4]
```diff
@@ -1,13 +1,14 @@
-cert-core>=2.1.10
-cert-schema>=3.0.0a9
-chainpoint>=0.0.2
+cert-core>=3.0.0b1
```
Comment (lemoustachiste) [inline requirements.txt:1]: publish v3 and update here

File: cert_issuer/blockchain_handlers/ethereum/connectors.py [hunk 5]
```diff
@@ -137,8 +137,8 @@ def get_balance(self, address):
         """
         Returns the balance in Wei.
         """
-        logging.info('Getting balance with EthereumRPCProvider')
         response = self.w3.eth.getBalance(account=address, block_identifier="latest")
+        logging.info('Getting balance with EthereumRPCProvider: %s', response)
```
Comment (lemoustachiste) [inline cert_issuer/blockchain_handlers/ethereum/connectors.py:141]: remove trailing log

File: requirements.txt (hunk 4 above)
Comment (lemoustachiste) [reply requirements.txt:1]: @fuerve to check if the branch exists somewhere

File: requirements.txt (hunk 3 above)
Comment (lemoustachiste) [reply requirements.txt:2]: @lemoustachiste to update the dependencies in cert-tools https://github.com/blockchain-certificates/cert-tools/blob/master/requirements.txt#L2

**Comment by lemoustachiste** [comment]
@lemoustachiste to update readme

File: .gitignore (hunk 1 above)
Comment (lemoustachiste) [reply .gitignore:23]: nothing critical it can stay

File: cert_issuer/blockchain_handlers/ethereum/connectors.py (hunk 5 above)
Comment (lemoustachiste) [reply cert_issuer/blockchain_handlers/ethereum/connectors.py:141]: not so useless

**Review by fuerve — APPROVED** [review]
Looks great.

**Pull request merged**